### ISSUE TITLE
problem with backticks in inline code at the start of a line

### ISSUE THREAD

**Issue by lucaswerkmeister** [rjeschke/txtmark#23]
Compare the result of the following markdown (use copy+paste – there’s a difference between 2. and 3.):

1.
```markdown
foo
`` `class Object` `` is a *meta literal*.
bar
```
2.
```markdown
foo
 `` `class Object` `` is a *meta literal*.
bar
```
3.
```markdown
foo
 `` `class Object` `` is a *meta literal*.
bar
```
(that’s a No-Break Space at the beginning of the second line) 
4.
```markdown
foo
other `` `class Object` `` is a *meta literal*.
bar
```
Here’s how the markdown is rendered by Github:

1.
>foo
`` `class Object` `` is a *meta literal*.
bar

2.
>foo
 `` `class Object` `` is a *meta literal*.
bar

3.
>foo
 `` `class Object` `` is a *meta literal*.
bar

4.
>foo
other `` `class Object` `` is a *meta literal*.
bar

(The line breaks are incorrect because [GFM](http://github.github.com/github-flavored-markdown/) always inserts line breaks for newlines.)

And here’s how the markdown is rendered by txtmark (as used by `ceylon doc`):

1.
>foo
`bar`

2.
>foo
`bar`

3.
>foo `` `class Object` `` is a *meta literal*. bar

4.
>foo other `` `class Object` `` is a *meta literal*. bar

That is, if a line starts with nothing other than spaces (meaning the identity of U+0020, not the Unicode category “Separator, Space [Zs]”, which also contains the No-Break Space U+00A0 as seen in 3.) and then a multi-backtick code block containing backticks ``` `` `code` with backticks` `` ```, then strange behavior occurs.

<sup>(originally reported as ceylon/ceylon-compiler#1553)</sup>

**Comment by rjeschke** [comment]
I will add a flag to the `Configuration` to enable/disable spaces in fenced code blocks. This will allow to fix this issue without destroying compatibility.

**Comment by rjeschke** [comment]
Fixed with a933bf5f891c3097b0d5bda84f157d0c876eeaea
